tests/functional/test_config.py

```diff
@@ -55,6 +55,32 @@ def test_ethereum_network_configs(config, temp_config):
         assert actual.rinkeby.block_time == 15
 
 
+def test_network_gas_limit_default(config):
+    eth_config = config.get_config("ethereum")
+
+    assert eth_config.rinkeby.gas_limit == "auto"
+    assert eth_config.local.gas_limit == "auto"
```

Comment:
In 0.6 or any future release, we may want to consider making `max` the default for `local`.
I think it will give speed improvements to running test suites.

Reply:
Good point, and what's funny is I thought previously I had local set to `max` but I can't seem to find trace of that now 🤔  it was before that pre-audit work so I think I lost a bit of track with the gear switching

I may need to revisit this for #957 but I'll see about including such changes in that PR if need be 👍

Reply:
If we want to make it use max now, that is probably fine. It should not affect anything really so maybe not a real breaking change